bin/xbps-rindex/xbps-rindex.1

```diff
@@ -56,6 +64,13 @@ Absolute path to the local repository is expected.
 .It Sy -c, --clean Ar /path/to/repository
 Removes obsolete entries found in the local repository.
 Absolute path to the local repository is expected.
+.It Sy -R, --remove Ar /path/to/repository /path/to/repository/binpkg.xbps ...
```

Comment:
dpes it expect one arg or two? there's something wrong, either in the docs or in the code.